src/pages/home/report/ReportActionItem.js

```diff
@@ -379,20 +379,27 @@ function ReportActionItem(props) {
                 />
             );
         } else if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW) {
-            children = (
-                <ReportPreview
-                    iouReportID={ReportActionsUtils.getIOUReportIDFromReportActionPreview(props.action)}
-                    chatReportID={props.report.reportID}
-                    policyID={props.report.policyID}
-                    containerStyles={props.displayAsGroup ? [] : [styles.mt2]}
-                    action={props.action}
-                    isHovered={hovered}
-                    contextMenuAnchor={popoverAnchorRef}
-                    checkIfContextMenuActive={toggleContextMenuFromActiveReportAction}
-                    isWhisper={isWhisper}
-                    transactionViolations={props.transactionViolations}
-                />
-            );
+            const iouReportID = ReportActionsUtils.getIOUReportIDFromReportActionPreview(props.action);
+            const iouReport = ReportUtils.getReport(iouReportID);
+            const isClosedReport = ReportUtils.isClosedExpenseReportWithNoExpenses(iouReport);
+            if (!isClosedReport) {
+                children = (
+                    <ReportPreview
+                        iouReportID={ReportActionsUtils.getIOUReportIDFromReportActionPreview(props.action)}
+                        chatReportID={props.report.reportID}
+                        policyID={props.report.policyID}
+                        containerStyles={props.displayAsGroup ? [] : [styles.mt2]}
+                        action={props.action}
+                        isHovered={hovered}
+                        contextMenuAnchor={popoverAnchorRef}
+                        checkIfContextMenuActive={toggleContextMenuFromActiveReportAction}
+                        isWhisper={isWhisper}
+                        transactionViolations={props.transactionViolations}
+                    />
+                );
+            } else {
```

Comment:
Let's go with an early return instead of the if/else pattern to stay consistent with the surrounding code